services/dy-dash/cc-0d-dash/src/cc-0d-dash/cc-0d.py

```diff
@@ -347,25 +347,156 @@ def create_graph_4():
     title="[Ca]<sub>i</sub> (\u00B5M)"
     axis_colums = [0,ynid[38]+1]
     plot_5 = data_frame_ty.filter(items=[data_frame_ty.columns[i] for i in axis_colums])
-    fig = create_graph(data_frame=plot_5, 
-                title=None, 
+    fig = create_graph(data_frame=plot_5,
+                title=None,
                 x_axis_title="time (sec)",
                 y_axis_title=title)
     return fig
 
 def create_graph_5():
+    # Ito
```

Comment:
Lots of copy&paste here, oder? :-)  Wouldn't it be better to do a single ``create_graph`` function and pass parameters?